WebContent/Navigation.jsp

```diff
@@ -0,0 +1,36 @@
+<% if (session.getAttribute("user") != null) { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSP file should use UTF-8 encoding](https://app.codacy.com/manual/cf18patricia.lamadrid/Sudoers-Academy/pullRequest?prid=5452086)